### PR TITLE
Update 125-fourier-series.md

### PR DESCRIPTION
**Pull request by lautibonet**
Fourier Challenge contribution

### CODE REVIEW

File: _CodingChallenges/125-fourier-series.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ videos:
     url: "/Tutorials/9-additional-topics/9.20-polar-coordinates"
   - title: "Fractal Spirograph"
     url: "/CodingChallenges/061-fractal-spirograph"
+
```
Comment (gruselhaus) [inline _CodingChallenges/125-fourier-series.md]:
```suggestion
```

File: _CodingChallenges/125-fourier-series.md [hunk 2]
```diff
@@ -25,6 +25,13 @@ videos:
     url: "/Tutorials/9-additional-topics/9.20-polar-coordinates"
   - title: "Fractal Spirograph"
     url: "/CodingChallenges/061-fractal-spirograph"
+  
+  contributions:
```
Comment (gruselhaus) [inline _CodingChallenges/125-fourier-series.md]:
```suggestion
contributions:
```

**Comment by gruselhaus** [comment]
Excellent work! 🖤 @shiffman may I merge?

**Comment by shiffman** [comment]
@gruselhaus yes, please feel free!

**Pull request merged**